virt/virt-4-9-release-notes.adoc

```diff
@@ -42,7 +42,14 @@ Red Hat is committed to replacing problematic language in our code, documentatio
 
 //CNV-8577 OpenShift Service Mesh
 
-//CNV-12858 SVVP previous 8914
+//CNV-12858
+* {VirtProductName} is certified in Microsoft’s Windows Server Virtualization Validation Program (SVVP) to run Windows Server workloads.
++
+The SVVP Certification applies to:
++
+** Red Hat Enterprise Linux CoreOS workers. In the Microsoft SVVP Catalog, they are named __Red Hat OpenShift Container Platform 4 on RHEL CoreOS__.
+** Intel and AMD CPUs.
+* The Containerized Data Importer (CDI) now uses the {product-title} xref:../networking/enable-cluster-wide-proxy.adoc#enable-cluster-wide-proxy[cluster-wide proxy configuration].
```

Comment:
In the last release note (cnv-4.8) we did not add the CDI version, do we need it?

Reply:
Whoops, that line is a hitchhiker from another item on the 4.8 RN. Will remove.